server/app/models/ApplicationModel.java

```diff
@@ -94,11 +94,6 @@ public ProgramModel getProgram() {
     return this.program;
   }
 
-  /** Returns the admin name of the program this application is associated with. */
-  public String getProgramName() {
-    return getProgram().getProgramDefinition().adminName();
```

Comment:
[Non blocking] clarification - In future, if we want to get the admin name of a program, best would be to inject the programRepository class and do a getProgramDefinition().getAdminName()?

Reply:
Yeah exactly! We can do `programRepository.getProgramDefinition(program).adminName()`. Basically, we just want to avoid calling methods from the Models directly as much as possible.